src/Nav/Nav.tsx

```diff
@@ -3,19 +3,39 @@ import AppBar from "@mui/material/AppBar";
 import Box from "@mui/material/Box";
 import Toolbar from "@mui/material/Toolbar";
 import Typography from "@mui/material/Typography";
-import Button from "@mui/material/Button";
 import NavMenu from "./NavMenu";
+import NavButton from "../components/navButton/NavButton";
+// import Button from "@mui/material/Button";
+// import LoginIcon from '@mui/icons-material/Login';
+
 
 export default function Nav() {
```

Comment:
We're using TypeScript! 
So a couple of things. 

The first is mainly a preference thing: 

You're declaring many new functions that return TSX (aka the typescript version of JSX ) code. JSX is a combination of HTML and JavaScript used by react that makes it very easy for us to write JS alongside our HTML. 

You're declaring them with regular Function syntax i.e. `function Nav() {...}`

However, we are using esLint, which checks our code for errors automatically to help us keep things clean.. And esLint prefers by default, or out of the box, to have functional components declared with **arrow function syntax.** Arrow Functions were introduced in a newer version of JS that most use today. They look like this:

`const Nav = () => {
    return (
     )
}

So we need to convert all of these to Arrow Functions. 

in case you didn't know, TypeScript is what's called a **superset** of JavaScript. That means that under-the-hood, TypeScript _is_ JavaScript, but it wraps it in "frosting" to give us some extra tools. Computers/Browsers don't know how to read TypeScript, so all TypeScript is actually compiled in JavaScript before anything is run on a browswer/computer. A functional component in TypeScript written as an arrow function/normal function compile to JS and look exactly the same -- it's purely for esLint.

Secondly, we _are_ using typescript, so we should type all of our functional components.

So the "final" nav component would look like this

`export default Nav = (): ReactElement  => {...}`